content/quickstart/java/metrics.md

```diff
@@ -13,8 +13,8 @@ class: "shadowed-image lightbox"
 - [Getting started](#getting-started)
 - [Enable Metrics](#enable-metrics)
     - [Import Packages](#import-metrics-packages)
-    - [Create Metrics](#create-metrics)
-    - [Create Tags](#create-tags)
+    - [Declare Metrics](#declare-metrics)
```

Comment:
How come the change to `Declare`? We are actually creating tags and metrics instead of declaring them. A declaration is just spelling out that some variable or function will be defined somewhere, this could massively cause confusion.

Reply:
I was going off of the quoted advice from offline feedback:

> I'd maybe call this something like "Create Variables for Metrics" or "Declare Metrics". "Create Metrics" can be construed as the actual instantiation of metric data.

Should I change it back to "Create"?